File: app/src/main/java/com/huhx0015/androidbooster/utils/DisplayUtils.java
package com.huhx0015.androidbooster.utils;

import android.content.Context;
import androidx.annotation.NonNull;

public class DisplayUtils {

    /** UTILITY METHODS ________________________________________________________________________ **/

    /**
     * getOrientation(): Retrieves the device's current orientation.
     *
     * Configuration.ORIENTATION_PORTRAIT
     * Configuration.ORIENTATION_LANDSCAPE
     * @param context
     * @return
     */
    public static int getOrientation(@NonNull Context context) {
        return context.getResources().getConfiguration().orientation;
    }
}

File: app/src/main/java/com/huhx0015/androidbooster/utils/DisplayUtils.kt
package com.huhx0015.androidbooster.utils

import android.content.Context

object DisplayUtils {

    /** UTILITY METHODS ________________________________________________________________________  */

    /**
     * getOrientation(): Retrieves the device's current orientation.
     *
     * Configuration.ORIENTATION_PORTRAIT
     * Configuration.ORIENTATION_LANDSCAPE
     * @param context
     * @return
     */
    fun getOrientation(context: Context): Int {
        return context.resources.configuration.orientation
    }
}
